eng/test-configuration.json

```diff
@@ -11,6 +11,7 @@
     {"testName": {"contains": "AddressRegistrationTests" }},
     {"testName": {"contains": "MaxRequestBufferSizeTests.LargeUpload" }},
     {"testName": {"contains": "ReaderThrowsResetExceptionOnInvalidBody" }},
+    {"failureMessage": "System.AggregateException : One or more errors occurred. (Operation did not succeed after 15 retries (Site is started but no worker process found) (Site is started but no worker process found)"},
```

Comment:
Not positive whether this should use `"contains"` and am asking in Dev WF channel. If this is wrong, I bet the line below is as well.

Reply:
Maybe just limit it to Operation did not succeed after 15 retries (Site is started but no worker process found)?

Reply:
Also note: I'm currently looking for the root cause of this error, as I believe there's something wrong with something in the assumptions in that shared code since as far as I can tell, those 15 retries never help when no worker process is found

Reply:
Yeah, this line was busted

Reply:
> Maybe just limit it to Operation did not succeed after 15 retries (Site is started but no worker process found)?

My thought was two of the 15 copies of the parenthesized block was enough to confirm the final failure wasn't something else. Would rather keep at least one extra here.

> Also note: I'm currently looking for the root cause of this error

Should I close this given that work❔

Reply:
> Should I close this given that work❔

I take that back because this PR also fixes the existing `"failure Message"` match. Would just remove the added line if that's your preference @HaoK